virtool/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "virtool"
+version = "0.1.0"
+description = ""
+authors = ["Aman Monga <aman260@outlook.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+
+[tool.poetry.dev-dependencies]
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
Remove this file.

I think you accidently created another Poetry project in the `virtool/virtool` folder.